### ISSUE TITLE
Redirect to GitHub Guide

### ISSUE THREAD

**Issue by loganfranken** [UCSantaBarbara/webguide#14]
At some point, we will need to redirect from http://www.ucsb.edu/webguide/ to http://ucsb-wsg.github.io/

**Comment by loganfranken** [comment]
Unless anyone disagrees, I think we can close this one: #20 is essentially a duplicate with a more fleshed out discussion.

**Comment by garster** [comment]
Yes, but I also think we should make sure a redirect is step up an the old http://www.ucsb.edu/webguide/ goes to what ever is done in #20. 

So I do see merit in this standalone issue.

**Comment by loganfranken** [comment]
@garster Oh, gotcha: #20 is for subdomain vs subdirectory and this one's for the actual redirect. Alright, yeah, I agree with you on keeping it then, yeah.

**Comment by tenken** [comment]
A request was placed with Public Affairs to put an Apache redirect. They consulted with NOC but need to wait until Kevin S. returns on July 7th to modify the vhost entry. Feedback pending.

**Comment by loganfranken** [comment]
This is working now, right? (Wahoo! BTW)

**Comment by tenken** [comment]
It is now. It had been implemented to redirect from old pages of the site, to the new subdomain, and was producing 404s. I had requested they dumb it down to send the user just to the homepage.

I didnt receive a notification it was fixed for the 2nd issue -- but it does appear to be fixed with a simple test of www.ucsb.edu/webguide/foo.html. I'm closing this issue.
